.github/workflows/extrinsic-ordering-check-from-bin.yml

```diff
@@ -33,35 +22,33 @@ jobs:
 
       - name: Prepare output and compare the metadata
         run: |
-          CHAIN=${CHAIN:-litmus}
           BASE_URL=wss://parachain-eu-0.staging.litentry.io
```

Comment:
This is incorrect if I understand it correctly, we'd need to compare the live chain and the image that is going to be released. Therefore we need to start the parachain network locally (or only the parachain image if that works) beforehand using the to-be-released docker image

You could refer to [polkadot](https://github.com/paritytech/polkadot/blob/9804117d3e676aace31ed39569a4e6aec75956de/.github/workflows/release-20_extrinsic-ordering-check-from-bin.yml#L60) or [acala](https://github.com/AcalaNetwork/Acala/blob/6334e0f5547bfc5aeb30a19a94730ad582cca593/.github/workflows/extrinsic-ordering-check-from-bin.yml#L50)